sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/DotCoverReportsAggregator.java

```diff
@@ -54,6 +55,8 @@ public void accept(File file, Coverage coverage) {
     }
 
     List<File> reportFiles = listReportFiles(folder);
+    LOG.debug("DotCover aggregator: collected '{}' report files to parse", reportFiles.size());
```

Comment:
dotCover website uses lower case letter at the beginning of sentence.
```suggestion
    LOG.debug("dotCover aggregator: collected '{}' report files to parse", reportFiles.size());
```